packages/cli/commons/src/utils/ux.ts

```diff
@@ -1,6 +1,10 @@
 import {CliUx} from '@oclif/core';
 import {red, green, magenta} from 'chalk';
 
+export function shouldUseColor() {
+  return process.platform !== 'win32' || Boolean(process.env['WT_PROFILE_ID']);
+}
```

Comment:
Not a very strong preference, but I would prefer if this was divided in a more verbal(?) kind of way.

Like:
```suggestion
export function shouldUseColor() {
  const isWindows = (process.platform === 'win32');
  if (isWindows) {
    const isWindowsTerminal = Boolean(process.env['WT_PROFILE_ID']);
    return isWindowsTerminal;
  }
  return true;
}
```

Reply:
```suggestion

functions isWindows() {
  return process.platform === 'win32'
}

function isWindowsTerminal() {
  return Boolean(process.env['WT_PROFILE_ID']);
}

export function shouldUseColor() {
  return !isWindows() || isWindowsTerminal();
}
```

Reply:
Similar to your second proposition, but I found the following more readable.